accounts.py

```diff
@@ -0,0 +1,10 @@
+accounts = {}
+
+
+def add_account(name, password):
+    """Function enables user to create an account"""
```

Comment:
I think you should adopt docstrings that describe your function better.
Great work. though.